src/app/pages/storage/storage.routing.ts

```diff
@@ -1,27 +1,30 @@
 import { ModuleWithProviders } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+import { TranslationsLoadedGuard } from 'app/core/guards/translations-loaded.guard';
 import { PoolsDashboardComponent } from 'app/pages/storage/components/pools-dashboard/pools-dashboard.component';
 import { ManagerComponent } from './components/manager/manager.component';
 
 export const routes: Routes = [
   {
     path: '',
-    data: { title: 'Storage' },
+    canActivate: [TranslationsLoadedGuard],
```

Comment:
This isn't necessary, as this guard is applied on a level higher in `app.routes.ts`.